citus-enterprise.spec

```diff
@@ -1,5 +1,5 @@
-%global pgmajorversion 10
-%global pgpackageversion 10
+%global pgmajorversion 11
```

Comment:
Do we still need pgmajorversion check at the line 67 ?

Reply:
I guess we don't. I'll remove that unnecessary check before merging.